fastify.d.ts

```diff
@@ -149,6 +149,7 @@ type TrustProxyFunction = (address: string, hop: number) => boolean
 declare module 'fastify-error' {
   interface FastifyError {
     validation?: ValidationResult[];
+    validationContext?: string;
```

Comment:
Is it not possible to define the values it can assume?

Reply:
I think it's safer to just type as a string instead of a union.